### PR TITLE
[proxy] account_resolver_test.go

### PR DESCRIPTION
**Pull request by IljaN**

### CODE REVIEW

File: proxy/pkg/middleware/account_resolver_test.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+	sut.ServeHTTP(rw, req)
+
+	token := req.Header.Get("x-access-token")
+	assert.NotEmpty(t, token)
+
+	assert.Contains(t, token, "eyJ")
+}
+
+func TestNSkipOnNoClaimst(t *testing.T) {
```
Comment (kulmann) [inline proxy/pkg/middleware/account_resolver_test.go:53]:
```suggestion
func TestNSkipOnNoClaims(t *testing.T) {
```

File: proxy/pkg/middleware/account_resolver_test.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+	}, nil)
+
+	req, rw := mockRequest(&oidc.StandardClaims{
+		Iss:   "https://idx.example.com",
+		Email: "foo@example.com",
+	})
+
+	sut.ServeHTTP(rw, req)
+
+	token := req.Header.Get("x-access-token")
```
Comment (kulmann) [inline proxy/pkg/middleware/account_resolver_test.go:29]: IMO makes sense to use the const from reva for the token header all over the place:
https://github.com/cs3org/reva/blob/314157fa14378d7e8deac6042cf193d11ebbb069/pkg/token/token.go#L29

File: proxy/pkg/middleware/account_resolver_test.go [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+	assert.Empty(t, token)
+	assert.Equal(t, http.StatusUnauthorized, rw.Code)
+}
+
+func TestInternalServerErrorOnMissingMailAndUSername(t *testing.T) {
```
Comment (kulmann) [inline proxy/pkg/middleware/account_resolver_test.go:92]:
```suggestion
func TestInternalServerErrorOnMissingMailAndUsername(t *testing.T) {
```

File: proxy/pkg/middleware/account_resolver_test.go [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+	sut := newMockAccountResolver(nil, backend.ErrAccountDisabled)
+	req, rw := mockRequest(&oidc.StandardClaims{
+		Iss: "https://idx.example.com",
+	})
+
+	sut.ServeHTTP(rw, req)
+
+	token := req.Header.Get("x-access-token")
+	assert.Empty(t, token)
+	assert.Equal(t, http.StatusInternalServerError, rw.Code)
```
Comment (kulmann) [inline proxy/pkg/middleware/account_resolver_test.go:102]: Why is a missing mail and username not a `http.StatusBadRequest`? 🤔

Comment (IljaN) [reply proxy/pkg/middleware/account_resolver_test.go:102]: I would like to takle this in a separate issue

Comment (kulmann) [reply proxy/pkg/middleware/account_resolver_test.go:103]: Ok, fine by me. Did you already create one? :-) If not, please do, so that we don't forget it.

**Review by kulmann — APPROVED** [review]

**Pull request merged**